scripts/release/publish-commands/publish-to-npm.js

```diff
@@ -19,10 +19,17 @@ const push = async ({cwd, dry, packages, version, tag}) => {
     throw new Error('The tag `latest` can only be used for stable versions.');
   }
 
+  // Pass two factor auth code if provided:
+  // https://docs.npmjs.com/getting-started/using-two-factor-authentication
+  const twoFactorAuth = opt != null ? `--opt ${opt}` : '';
```

Comment:
otp?

Reply:
Gah! I read it, multiple times, as "opt" 😦 but you're right.